guess_the_words/function.py

```diff
@@ -0,0 +1,161 @@
+import json
+import string
+import random
+import datetime
+
+
+def create_list():
+    with open('words.txt', 'r') as file:
+        list_words = file.readlines()
+        for posit, word in enumerate(list_words):
+            list_words[posit] = word.strip()
```

Comment:
```suggestion
            list_words[posit] = word.strip().lower()
```

Reply:
I agree with you :) I forgot aplly method .lower() in some functions (((